File: Programmers/lv1/lv1_수박수박수박수박수박수.java
https://programmers.co.kr/learn/courses/30/lessons/12922

class Solution {
    public String solution(int n) {
        String answer = "";
        for(int i = 1; i <= n; i++){
            answer += (i%2 != 0)? "¼ö" : "¹Ú";
        }
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_수박수박수박수박수박수.cs
https://programmers.co.kr/learn/courses/30/lessons/12922

public class Solution
{
    public string solution(int n)
    {
        string answer = "";
        for (int i = 0; i < n; i++)
            answer += (i % 2 == 0) ? "¼ö" : "¹Ú";
        return answer;
    }
}